modules/tensor_mechanics/include/materials/ComputeRotatedElasticityTensorBase.h

```diff
@@ -0,0 +1,24 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#ifndef COMPUTEROTATEDELASTICITYTENSORBASE_H
+#define COMPUTEROTATEDELASTICITYTENSORBASE_H
+
+#include "ComputeElasticityTensorBase.h"
+
+/**
+ * ComputeRotatedElasticityTensorBase defines an elasticity tensor material object with a given base name.
```

Comment:
We should update this comment as well to mention that this is an intermediate base class that adds Euler angles